applications/CompressiblePotentialFlowApplication/custom_conditions/potential_wall_condition.h

```diff
@@ -292,7 +292,7 @@ class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) PotentialWallCondition
     ///@name Private Operators
     ///@{
 
-    inline ElementPointerType pGetElement() const;
+    inline GlobalPointer<Element> pGetElement();
```

Comment:
@RiccardoRossi this is what I was referring to when I had to remove the `const` definition (see cpp).